communications.md

```diff
@@ -2,95 +2,62 @@
-
-#### - Internal & External Communication Tool: Gmail -
-
-Minimize Distractions: A busy email inbox is a contributing factor to interruptions. Instead of striving to achieve "Inbox Zero", try to be as close as possible to "Everything Zero". This means diligently clear or defer any information that is not work-related. You may do the following:
-
-- Pin/star items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Optionally snooze items that need future attention. Archive them as soon as they're done, or no longer needed.
-- Archive the rest.
-
-Opt-out: Use filters to avoid receiving emails sent to a shared email address that you are not interested in.
 
 #### - Project Management & Product Issue Documentation: Asana -
 
-Opt-in: Join projects that you are interested in or obligated to.
-
-Minimize Distractions: Similar to Slack, instead of striving to achieve "Notification Zero", try to be as close as possible to "Everything Zero". Defer any information that is not work-related
-
-Documentation - Transparency: Provide as much detail as possible for other team members for context and opt-in to a task or issue if necessary.
-
-Opt-out: Leave projects that you are no longer interested in or obligated to.
-
-## Second line of Communication: Meeting
+Our Asana workspace is commonly used for project management e.g task allocation, task state (in progress or completed), and any other process we deem it fit to be used for (e.g Office Availability, calender scheduling, etc.)
+Opt-in to join projects/tasks that you are interested in or obligated to.
 
-If a meeting is necessary, budget the amount of time needed appropriately. Each meeting is the collective productivity value of all its participants. Consider each participant to be on average contributing $200/hr; it is always good to judge the need/value of each meeting against the economic value of the meeting (what other productive tasks could these participants be doing?).
 
-#### - Meetings & Events Tool: Google Calendar -
+#### Meetings
+If a meeting is necessary, we have a meeting space that can be pre-booked and used appropriately. Please be mindful of the duration time as we presently have limited private space for meetings and others might be waiting in line.
 
-Opt-in: Subscribe to calendars you are interested in or obligated to.
+###### Events Tool: Google Calendar -
+Opt-in to subscribe to calendars you are interested in or obligated to.
+You can set up meetings by using event invitations and book one of the available rooms in the google invitation to avoid disappointment.
 
-Set up meetings: Use event invitations to set up a meeting and book one of the available rooms in the google invitation to avoid disappointment.
+###### Preparing for & During Meetings: 
+To ensure that meetings are productive and are respectful of everyone's time and productivity, feel free to try out the steps below:
```

Comment:
Changing `you may` to `feel free` doesn't really make this more, or less suggestive, to be honest.

Reply:
you **may**: meaning - 1. expressing possibilities, 2. used when admitting that something is so before making another, more important point

**feel free**: meaning - have no hesitation or shyness (often used as an invitation or for reassurance)

This words could be interpreted as similar or different but it can set a tone to a conversation. Regardless, it can be reviewed and left as "may" or "feel free" (this is minor). The main focus is that we address point of being subtle.

Reply:
Fair enough.